mockSchemaRegistryClient_test.go

```diff
@@ -2,82 +2,80 @@ package srclient
 
 import (
 	"fmt"
-	"github.com/stretchr/testify/assert"
 	"sort"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 /*
 We are going to test the client meant to be used for testing
- */
+*/
 
 var srClient MockSchemaRegistryClient
 var schema = "{  " +
-"\"type\": \"record\",  " +
-"\"namespace\": \"com.mycorp.mynamespace\",  " +
-"\"name\": \"value_cdc_fake_2\", " +
-"\"doc\": \"Sample schema to help you get started.\", " +
-"\"fields\": [    " +
-"{   " +
-"\"name\": \"aField\"," +
-"\"type\": \"int\", " +
-"\"doc\": \"The int type is a 32-bit signed integer.\"   " +
-"}" +
-"]" +
-"}"
+	"\"type\": \"record\",  " +
```

Comment:
Probably more preferable to use a string literal for defining these. https://golang.org/ref/spec#String_literals

Reply:
Thank you for your reply, I'm re-adapting this work at [v1.0.0 branch](https://github.com/AtakanColak/srclient/tree/v1.0.0-alpha) though it is not near completion yet.